independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/rewrite/QuarkusUpdatesRepository.java

```diff
@@ -56,7 +56,7 @@ public static FetchResult fetchRecipes(MessageWriter log, MavenArtifactResolver
             final Artifact artifact = artifactResolver.resolve(DependencyUtils.toArtifact(gav)).getArtifact();
             final ResourceLoader resourceLoader = ResourceLoaders.resolveFileResourceLoader(
                     artifact.getFile());
-            final List<String> recipes = fetchRecipesAsList(resourceLoader, "quarkus-updates", recipeDirectoryNames);
+            final List<String[]> recipes = fetchRecipesAsList(resourceLoader, "quarkus-updates", recipeDirectoryNames);
```

Comment:
I had another look at this and could we use a `record` instead of an obscure array? It's very cryptic and adding a record won't take much time.